src/index.ts

```diff
@@ -83,8 +85,8 @@ export interface Field {
 
   columnLength?: number | null
   charset?: number | null
+  decimals?: number
```

Comment:
What is this change for? And why was `columnType?` removed instead? Can we just back out this change in this PR and follow up if it's something different and unrelated?

Reply:
I would argue, if you're going to make changes here, we might as well match the proto spec: https://buf.build/planetscale/vitess/docs/9c148f260a264212bb8f6e1343e87685:vitess.query.v16#vitess.query.v16.Field

And I'd see no reason to remove columnType, because it's in the proto.

If anything, adding decimal and keeping columnType.